app/models/local_authority_postcode.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: local_authority_postcodes
+#
+#  gss_code   :string           not null
+#  postcode   :string           not null
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+# Indexes
+#
+#  index_local_authority_postcodes_on_gss_code_and_postcode  (gss_code,postcode) UNIQUE
+#  index_local_authority_postcodes_on_postcode_and_gss_code  (postcode,gss_code) UNIQUE
+#
+class LocalAuthorityPostcode < ApplicationRecord
```

Comment:
```suggestion
LocalAuthority::Postcode
```

It could be nice to make this a nested class, since it's so closely related to the `LocalAuthority` model and helps to keep `app/models` well organised.

Reply:
That makes sense - I did find the name LocalAuthorityPostcode a bit of a mouthful, but couldn't think of a better one, so I will go with this 👍